### ISSUE TITLE
Need an Effective Document Display

### ISSUE THREAD

**Issue by mmangione** [Uzay-G/espial#12]
We should be able to click on a node and see the document in an in-browser render. We should also highlight specific words or content that links to other things. Like a document with a ton of clickable highlighted areas. It would also help to have a synopsis of the document, its links, and the key concepts and their links.

**Comment by Uzay-G** [comment]
Yeah I agree that this would be really nice, will put it on the list, but I don't have much time for the project right now.
